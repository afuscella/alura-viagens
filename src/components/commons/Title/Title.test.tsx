import React from 'react';
import { render } from '../../../infra/tests/test-utils';

import { Title } from './index';

describe('<Title />', () => {
  test('render component', () => {
    const { container } = render(
      <Title>
        Test
      </Title>,
    );

    expect(container).not.toBeNull();
    expect(container).toMatchSnapshot();
  });
});
