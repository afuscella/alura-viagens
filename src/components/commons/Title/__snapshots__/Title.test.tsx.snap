// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> render component 1`] = `
.c0 {
  color: red;
  font-size: 50px;
}

<div>
  <span
    class="c0"
  >
    Test
  </span>
</div>
`;
